PUBLISHING.md

```diff
@@ -0,0 +1,37 @@
+# Publishing
+
+This is how we currently publish images to dockerhub
+
+## Steps
+
+1. Create your auth.json either manually or using `docker login`
+
+```
+~$ echo -n 'sarcasticadmin:mysupersecretpassword' | base64
+c2FyY2FzdGljYWRtaW46bXlzdXBlcnNlY3JldHBhc3N3b3Jk
+
+~$ cat << EOF > auth.json
+{
+        "auths": {
+                "https://index.docker.io/v1/": {
+                        "auth": "c2FyY2FzdGljYWRtaW46bXlzdXBlcnNlY3JldHBhc3N3b3Jk"
+                }
+        }
+}
+```
+
+2. If done manually also `export REGISTRY_AUTH_FILE=$(pwd)/auth.json`
```

Comment:
What do you think about adding this into the code block above since all of it is part of the manual way?